config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/CreepingDread-AAAAAAAAAAAAAAAAAAAHMg==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Creepers can cause the worst problems. \n\nDrowning Creeper - when it explodes, it surrounds you in a ball of cobble filled with water. Not only do you have to escape, but the cobble is infested with silverfish! And those silverfish are all infernal as well.  The best way to deal with it is to first escape up or down.  If you have to clear the cobble ball, first clear the water. Surrounding the cobble with a dirt moat will keep the silverfish inside and unable to hide. Put down punjis, quicksand, or lava, and/or spectral glass. Make a wall with a 1x1 window and use that to break the cobbles - fishing silverfish are especially dangerous.\n\nEnder Creeper - Randomly teleports you a short distance - possibly deep underground. "
+      "desc:8": "Creepers can make you have a baaaad time. \n\nDrowning Creeper - when it explodes, it surrounds you in a ball of cobblestone filled with water. Not only do you have to escape, but the cobble is infested with silverfish! And those silverfish are all infernal!  The best way to deal with it is to first escape up or down.  If you have to clear the cobble ball, first clear the water. Surrounding the cobble with a dirt moat will keep the silverfish inside and unable to hide. Put down punjis, quicksand, or lava, and/or spectral glass. Make a wall with a 1x1 window and use that to break the cobbles - fishing silverfish is especially dangerous.\n\nEnder Creeper - Randomly teleports you a short distance - possibly deep underground. "
```

Comment:
if you change cobble to cobblestone, do so consistently at least. or maybe not at all

Reply:
Thank you! Fixed.